<?xml version="1.0"?>
<launch>

    <!--Run rviz-->
    <node pkg = "rviz" name = "rviz" type = "rviz" args="-d $(find robotsim)/rviz/drone.rviz">
    </node>
    

    <!--Run the node that will generate an example path-->
    <node pkg = "robotsim" name = "robot" type = "drone_sim.py" args="" output="screen">
        <rosparam command="load" file="$(find robotsim)/config/drone_sim.yaml" />
    </node>


    <!--Run the node that will generate an example path-->
    <node pkg = "examples" name = "path_generator" type = "example_path.py" args="" output="screen">
        <rosparam command="load" file="$(find examples)/config/pub_path.yaml" />
    </node>


    <!-- Run the node that controls the robot with vector fields -->
    <node pkg="quad_robot" name="quad_node" type="quad_node.py" args="" output="screen">
        <rosparam command="load" file="$(find quad_robot)/config/quad_params.yaml" />
    </node>

</launch>


